layouts/topnav/styled.ts

```diff
@@ -0,0 +1,81 @@
+import { Dots } from '@space-metaverse-ag/space-ui/icons'
+import Link from 'next/link'
+import styled from 'styled-components'
+
+const Logo = styled(Link)`
+  padding: 1.25rem 1.5rem;
+  display: flex;
+  border-right: ${({ theme }) => `1px solid ${theme.colors.dark[200]}`};
+`
+
+const Route = styled(Link)`
```

Comment:
This will most likely be an `<a>` tag since we will be redirecting outside the app to: www.app.tryspace.com
We could use 2 components like: `InternalRoute` using Link and `ExternalRoute` using <a> tag.
Then inside the topnav options, could have an object like:

```
const navItems = {
    {
        isInternal: true | false
        link: "/purchases" | "https://app.tryspace.com/create-space"
    },
}

navItems.map() => 
{ item.isInternal ? <InternalRoute href={item.url} /> : <ExternalRoute href={item.url} /> }
```

Reply:
Combined we can keep the Link tag and add a property to change the target